### PR TITLE
Ldanzinger/popup view

### PR DESCRIPTION
**Pull request by ldanzinger**
@michael-tims please review and merge. @tdunn please review the doc

cc @JamesMBallard

### CODE REVIEW

File: Examples/PopupExample.qml [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+            }
+        }
+    }
+
+    ComboBox {
+        id: displayModeComboBox
+        anchors {
+            left: parent.left
+            top: parent.top
+            margins: 5
```
Comment (michael-tims) [inline Examples/PopupExample.qml]: scaleFactor

File: Examples/PopupExample.qml [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+    Dialog {
+        id: popupAsDialog
+        standardButtons: StandardButton.Close
+        modality: Qt.NonModal
+
+        contentItem: PopupView {
+            id: popupViewDialog
+            implicitHeight: 400
+            implicitWidth: 400
```
Comment (michael-tims) [inline Examples/PopupExample.qml]: Do `implicitHeight` and `implicitWidth` require scaleFactor?  Another below.

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+        Column {
+            id: heading
+            anchors {
+                left: parent.left
+                right: parent.right
+                top: parent.top
+                margins: 10 * scaleFactor
+            }
+            spacing: 10
```
Comment (michael-tims) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml]: scaleFactor

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                bottom: parent.bottom
+                margins: 10 * scaleFactor
+            }
+            contentHeight: popupColumn.height
+            clip: true
+
+            Column {
+                id: popupColumn
+                width: parent.width
+                spacing: 10
```
Comment (michael-tims) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml]: scaleFactor

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+                clip: true
+
+                // display of attributes
+                Column {
+                    spacing: 5
```
Comment (michael-tims) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml]: scaleFactor

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+                    width: parent.width
+
+                    Repeater {
+                        model: displayedFields
+
+                        Row {
+                            anchors.left: parent.left
+                            clip: true
+                            spacing: 5
```
Comment (michael-tims) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml]: scaleFactor

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+                                font.pixelSize: 14 * scaleFactor
+                                color: attributeValueTextColorInternal
+                            }
+                        }
+                    }
+                }
+
+                // display of attachments
+                Column {
+                    spacing: 5
```
Comment (michael-tims) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml]: scaleFactor

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+                        renderType: Text.NativeRendering
+                    }
+
+                    Repeater {
+                        model: attachments
+
+                        Row {
+                            anchors.left: parent.left
+                            clip: true
+                            spacing: 5
```
Comment (michael-tims) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupViewBase.qml]: scaleFactor

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupView.qml [hunk 9]
```diff
@@ -0,0 +1,281 @@
...
+    \brief A view for displaying and editing information about a feature.
+
+    A PopupView can be used to display information for any type that
+    implements the PopupSource interface. For example, FeatureLayer
+    implements PopupSource. This means that it has a PopupDefinition,
+    which defines how the Popup should look for any features in that
+    layer.
+
+    An example workflow for displaying a PopupView for a feature in a
+    Feature Layer would be:
```
Comment (tdunn) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupView.qml]: Feature Layer > FeatureLayer

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupView.qml [hunk 10]
```diff
@@ -0,0 +1,281 @@
...
+        \brief The attribute value color of the PopupView.
+
+        The default color is \c "4f4f4f".
+    */
+    property color attributeValueTextColor: "#4f4f4f"
+
+    /*!
+        \brief The animation duration for the slideHorizontal and slideVertical methods.
+
+        The default duration is \c 250.
```
Comment (tdunn) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/PopupView.qml]: 250 what?

**Comment by tdunn** [comment]
Doc review is done. Very nice.

**Comment by ldanzinger** [comment]
thanks @michael-tims @tdunn. Updated now.

**Comment by michael-tims** [comment]
All good.

**Pull request merged**